MSAL/src/native_auth/public/state_machine/delegate/SignUpDelegates.swift

```diff
@@ -142,18 +149,21 @@ public protocol SignUpAttributesRequiredDelegate {
     @MainActor func onSignUpAttributesRequiredError(error: AttributesRequiredError)
 
     /// Notifies the delegate that there are some required attributes to be sent.
+    /// - Note: If a flow requires attributes but this optional method is not implemented, then ``onSignUpAttributesRequiredError(error:)`` will be called.
     /// - Parameters:
     ///     - attributes:  List of required attributes.
     ///     - newState: An object representing the new state of the flow with follow on methods.
-    @MainActor func onSignUpAttributesRequired(attributes: [MSALNativeAuthRequiredAttributes], newState: SignUpAttributesRequiredState)
+    @MainActor @objc optional func onSignUpAttributesRequired(attributes: [MSALNativeAuthRequiredAttributes], newState: SignUpAttributesRequiredState)
 
     /// Notifies the delegate that invalid attributes were sent.
+    /// - Note: If a flow requires attributes but this optional method is not implemented, then ``onSignUpAttributesRequiredError(error:)`` will be called.
```

Comment:
The comment says is attributes are required, but the method is for attributes invalid. As said before, prefer to use the generic comment about the optional method being required.

Reply:
This is now fixed